src/get_family_names.py

```diff
@@ -70,13 +72,23 @@ def flatten_parents(df):
 
 
 def get_all_congress_people_ids():
+    print('Fetching all congresspeople ids', end='\r')
     ids_series = [read_csv(name)['congressperson_id']
                   for name in ['current-year', 'last-year', 'previous-years']]
     return list(pd.concat(ids_series).unique())
 
 
+def find_latest_date():
+    date_regex = re.compile('\d{4}-\d{2}-\d{2}')
+
+    matches = (date_regex.findall(f) for f in os.listdir(DATA_DIR))
+    dates = (l[0] for l in matches if l)
```

Comment:
During the grupy-rp sprint, I had this idea: we could fallback to older dates in case the file doesn't exists, so the method find_newest_file would get the dates from newest to oldest returning the newest file.

```python
def find_newest_file(name):
    """
    Assuming that the files will be in the form of :
    yyyy-mm-dd-type_of_file.xz we can try to find the newest file
    based on the date, but if the file doesn't exist fallback to another
    date until all dates are exhausted
    """
    date_regex = re.compile('\d{4}-\d{2}-\d{2}')

    matches = (date_regex.findall(f) for f in os.listdir(DATA_DIR))
    dates = sorted(set([l[0] for l in matches if l]), reverse=True)
    for file_date in dates:
        current_filename = 'data/{}-{}.xz'.format(date, name)
        if os.path.isfile(current_filename):
            return current_filename

    return None
```

Then the read_csv function would need to be modified for something like this:

```python
def read_csv(name):
    filename = find_newest_file(name)
    if (filename is None):
        raise TypeError('could not find the \
                specified filename: {}'.format(filename))

    return pd.read_csv('data/{}-{}.xz'.format(date, name),
                       parse_dates=[16],
                       dtype={'document_id': np.str,
                              'congressperson_id': np.str,
                              'congressperson_document': np.str,
                              'term_id': np.str,
                              'cnpj_cpf': np.str,
                              'reimbursement_number': np.str})
```